hwdb.d/60-sensor.hwdb

```diff
@@ -156,6 +156,9 @@ sensor:modalias:acpi:INVN6500*:dmi:*svnASUSTeK*:*pnT100TAS:*
 sensor:modalias:acpi:INVN6500*:dmi:*svnASUSTeK*:pnT200TA:*
  ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
 
+sensor:modalias:platform:HID-SENSOR-200073*:dmi:*svn*ASUSTeK*:pnT103HAF:*
```

Comment:
Could you add a short comment about the model name ?
Unfortunately, there is no comment for ASUS models, but please do like for other makers, e.g. see entries for Archos in the above.